### PR TITLE
vulkanscenegraph: new package

### PR DESCRIPTION
**Pull request by podsvirov**

### CODE REVIEW

File: mingw-w64-vsg/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Maintainer: Konstantin Podsvirov <konstantin@podsvirov.pro>
+
+_fullname=VulkanSceneGraph
+_realname=vsg
```
Comment (MehdiChinoune) [inline mingw-w64-vsg/PKGBUILD:4]:
```suggestion
_realname=vulkanscenegraph
```
see: https://aur.archlinux.org/packages/vulkanscenegraph

File: mingw-w64-vsg/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+pkgver=1.1.3
+pkgrel=1
+pkgdesc="Vulkan & C++17 based Scene Graph Project (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url='http://www.vulkanscenegraph.org/'
+msys2_repository_url='https://github.com/vsg-dev/VulkanSceneGraph'
+license=('spdx:MIT')
+depends=("${MINGW_PACKAGE_PREFIX}-vulkan-loader"
+         "${MINGW_PACKAGE_PREFIX}-libwinpthread-git")
```
Comment (MehdiChinoune) [inline mingw-w64-vsg/PKGBUILD:17]:
```suggestion
depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
         "${MINGW_PACKAGE_PREFIX}-vulkan-loader"
         "${MINGW_PACKAGE_PREFIX}-libwinpthread-git")
```
see runtime dependencies in CI

File: mingw-w64-vsg/PKGBUILD (hunk 1 above)
Comment (podsvirov) [reply mingw-w64-vsg/PKGBUILD:4]: Overview:
- [vulkanscenegraph](https://repology.org/project/vulkanscenegraph/versions) (AUR)
- [vsg](https://repology.org/project/vsg/versions) (vcpkg, conan)
- [libvsg](https://salsa.debian.org/games-team/vulkanscenegraph/-/blob/master/debian/control?ref_type=heads) (debian)

The CMake package named `vsg` (target `vsg::vsg`).
Header files in `vsg` subfolder.
All classes in `vsg` namespace.

Why we can't use short name as in many cases?

Comment (MehdiChinoune) [reply mingw-w64-vsg/PKGBUILD:4]: We follow Arch naming in general.

**Comment by ognevny** [comment]
replace `vsg` with the $_realname in package_*()

**Comment by podsvirov** [comment]
> replace `vsg` with the $_realname in package_*()

Oh, your right!

File: mingw-w64-vsgxchange/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url='http://www.vulkanscenegraph.org/'
+msys2_repository_url='https://github.com/vsg-dev/vsgXchange'
+license=('spdx:MIT')
+depends=("${MINGW_PACKAGE_PREFIX}-vulkanscenegraph"
+         "${MINGW_PACKAGE_PREFIX}-freetype"
+         "${MINGW_PACKAGE_PREFIX}-gdal"
+         "${MINGW_PACKAGE_PREFIX}-assimp"
+         "${MINGW_PACKAGE_PREFIX}-openexr")
```
Comment (MehdiChinoune) [inline mingw-w64-vsgxchange/PKGBUILD:19]:
```suggestion
depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
         "${MINGW_PACKAGE_PREFIX}-assimp"
         "${MINGW_PACKAGE_PREFIX}-curl"
         "${MINGW_PACKAGE_PREFIX}-freetype"
         "${MINGW_PACKAGE_PREFIX}-gdal"
         "${MINGW_PACKAGE_PREFIX}-openexr"
         "${MINGW_PACKAGE_PREFIX}-vulkan-loader"
         "${MINGW_PACKAGE_PREFIX}-vulkanscenegraph")
```

File: mingw-w64-vsgxchange/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-cc"
+             "${MINGW_PACKAGE_PREFIX}-curl"
```
Comment (MehdiChinoune) [inline mingw-w64-vsgxchange/PKGBUILD:23]:
```suggestion
```

**Pull request merged**